conf/routes

```diff
@@ -11,11 +11,11 @@ GET            /assets/*file                controllers.Assets.versioned(path="/
 GET            /                            controllers.Giraffe.contributeRedirect
 GET            /healthcheck                 controllers.Healthcheck.healthcheck
 
-GET            /uk/                         controllers.Giraffe.contributeUK
-GET            /us/                         controllers.Giraffe.contributeUSA
-GET            /au/                         controllers.Giraffe.contributeAustralia
-GET            /eu/                         controllers.Giraffe.contributeEurope
-GET            /:countryGroup/              controllers.Giraffe.contribute(countryGroup: com.gu.i18n.CountryGroup)
+GET            /uk                         controllers.Giraffe.contributeUK
+GET            /us                         controllers.Giraffe.contributeUSA
+GET            /au                         controllers.Giraffe.contributeAustralia
+GET            /eu                         controllers.Giraffe.contributeEurope
+GET            /:countryGroup              controllers.Giraffe.contribute(countryGroup: com.gu.i18n.CountryGroup)
 
```

Comment:
This might be a stupid question but why do we need to explicitly set routes for uk, us au and eu if we have /:countryGroup?